app/components/shared/widgets/Widget.tsx

```diff
@@ -297,15 +293,7 @@ class Widget<Configuration> extends Component<WidgetProps<Configuration>, Widget
                             ))}
                     </div>
 
-                    {hasError && (
-                        <ErrorMessage
-                            autoHide={false}
-                            error={i18n.t(
-                                'widget.renderError',
-                                'Cannot render widget. Check browser console for details.'
-                            )}
-                        />
-                    )}
+                    {hasError && <ErrorMessage autoHide={false} error={i18n.t('widget.unexpectedError')} />}
```

Comment:
Shouldn't we have here:
```suggestion
                    {hasError && <ErrorMessage autoHide={false} header={i18n.t('widget.renderError.title')} error={i18n.t('widget.renderError.content')} />}
```
? 
That way we may probably remove `widget.unexpectedError` key, am I right?

Reply:
Yup
Necessary changes are coming right away 🚀

Reply:
Changes has been provided 🎉